docs/man/docker-create.1.md

```diff
@@ -61,9 +62,13 @@ IMAGE [COMMAND] [ARG...]
 **--cidfile**=""
    Write the container ID to the file
 
-**--cpuset**=""
+**--cpuset-cpus**=""
    CPUs in which to allow execution (0-3, 0,1)
 
+**--cpuset-mems**=""
+   MEMs in which to allow execution (0-3, 0,1), MEMs means memory nodes,
```

Comment:
Please simplify and remove run-on: "Memory nodes (MEMs) in which to allow execution (0-3, 0, 1). Only effective on NUMA systems."